src/Void.Core/BufferList.fs

```diff
@@ -72,10 +83,33 @@ module BufferList =
         | CoreCommand.WriteBufferToPath (bufferId, path) ->
             writeBufferToPath bufferList bufferId path
         | _ -> 
-            (bufferList, noMessage)
+            bufferList, noMessage
+
+    let private package bufferId message =
+        {
+            BufferId = bufferId
+            Message = message
+        }
+
+    let handleGetBufferContentsRequest bufferList envelope =
+        let buffers = (!bufferList).List

```

Comment:
Confusingly enough, it means dereference the `bufferList` reference; i.e. `*bufferList` in C++ (if I recall my C++). We are using mutable state here, not yet having figured out how to use monads. :)